addons/account/views/account_move_views.xml

```diff
@@ -1500,6 +1500,7 @@
                         <separator/>
                         <filter string="Invoice Date" name="invoicedate" context="{'group_by': 'invoice_date'}"/>
                         <filter string="Due Date" name="duedate" context="{'group_by': 'invoice_date_due'}"/>
+                        <filter string="Accounting Date" name="accountingdate" context="{'group_by': 'date'}"/>
```

Comment:
I know that the two filters above don't have a "_" in the name attribute, but in general we prefer to put it instead of having a super long name. So since it's a new filter let's make it pretty and put "accounting_date" instead of "accountingdate" ahah